code/modules/atmospherics/machinery/components/machines/crystallizer.dm

```diff
@@ -0,0 +1,332 @@
+#define MIN_PROGRESS_AMOUNT 				3
```

Comment:
Standard indentation for defines here and everywhere else please.

Reply:
ah yes, i made this before the changes to the contribution.dm